### PR TITLE
Add optional on-demand e2e-aws job for openshift/kubernetes

### PR DESCRIPTION
**Pull request by marun**
It's important to be able to run the default e2e suite on aws when diagnosing aws-specific features and when qualifying a major rebase.

Also, switch to using the loki workflow.

### CODE REVIEW

**Comment by marun** [comment]
Given that the e2e-aws job is optional, it's rehearsal failing should not preclude merge. It's likely that there are latent compatibility issues with aws from the 1.19 rebase.

**Comment by marun** [comment]
@stevekuznetsov RPM server issues appear to be blocking rehearsal execution.

File: ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml [hunk 1]
```diff
@@ -197,10 +197,14 @@ tests:
...
     workflow: openshift-kubernetes-e2e-gcp
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-loki
 - as: e2e-gcp
   steps:
     cluster_profile: gcp
-    workflow: openshift-e2e-gcp
+    workflow: openshift-e2e-gcp-loki
```
Comment (sttts) [inline ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml:207]: why is loki removed? @ingvagabund

Comment (ingvagabund) [reply ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml:207]: It has never been in openshift/kubernetes repo yet

**Pull request merged**